src/style.css

```diff
@@ -0,0 +1,56 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.block--1,
```

Comment:
You should remove blocks because it's not necessary here
and add it to your `stars_star`
```
background-image: url("./images/star.svg");
  background-repeat: no-repeat;
  background-position: center;
```